ush/python/pygfs/task/atmens_analysis.py

```diff
@@ -207,16 +311,18 @@ def finalize(self: Analysis) -> None:
                 diaggzip = f"{diagfile}.gz"
                 archive.add(diaggzip, arcname=os.path.basename(diaggzip))
 
+        # get list of yamls to cop to ROTDIR
+        yamls = glob.glob(os.path.join(self.task_config.DATA, '*atmens*yaml'))
+
         # copy full YAML from executable to ROTDIR
-        logger.info(f"Copying {self.task_config.jedi_yaml} to {self.task_config.COM_ATMOS_ANALYSIS_ENS}")
-        src = os.path.join(self.task_config.DATA, f"{self.task_config.RUN}.t{self.task_config.cyc:02d}z.atmens.yaml")
-        dest = os.path.join(self.task_config.COM_ATMOS_ANALYSIS_ENS, f"{self.task_config.RUN}.t{self.task_config.cyc:02d}z.atmens.yaml")
-        logger.debug(f"Copying {src} to {dest}")
-        yaml_copy = {
-            'mkdir': [self.task_config.COM_ATMOS_ANALYSIS_ENS],
-            'copy': [[src, dest]]
-        }
-        FileHandler(yaml_copy).sync()
+        for src in yamls:
+            logger.info(f"Copying {src} to {self.task_config.COM_ATMOS_ANALYSIS_ENS}")
+            dest = os.path.join(self.task_config.COM_ATMOS_ANALYSIS_ENS, os.path.basename(src))
+            logger.debug(f"Copying {src} to {dest}")
+            yaml_copy = {
+                'copy': [[src, dest]]
+            }
+            FileHandler(yaml_copy).sync()
```

Comment:
This isn't new, but why are the yaml being copied to COM?

Reply:
The yamls used to run JEDI jobs are copied to `COM` for documentation and reproducibility reasons.  JEDI configurations are driven by dynamically rendered yamls, not scripts or fixed input yamls.  While the job log file captures an echo of the input yaml, the log file format is not easy to read.  By copying the input yaml to `COM` and then archiving the yaml, developers should, in principle, be able to reproduce any given job.

Reply:
Okay, we might want these in `COM_CONF` instead then. Will think about it (doesn't have to be this PR).